src/status_im/events.cljs

```diff
@@ -481,3 +487,25 @@
  :notifications.callback/request-notifications-permissions-denied
  (fn [cofx _]
    (accounts/show-mainnet-is-default-alert cofx)))
+
+;; hardwallet module
+
+(handlers/register-handler-fx
+ :hardwallet.callback/set-nfc-support
```

Comment:
same for the 4 following events

Reply:
functions names are fine they should reflect what will happen, but events must reflect what happened not what will happen